commit 47e7676e7e1d39bf8bcd5fa623698f383a60712e
Author: wm4 <wm4@nowhere>
Date:   Sat Apr 23 17:24:03 2016 +0200

player: force VO reconfig when unselecting video track

Switches to a black window if --force-window is used while coverart
"video" is playing.
---

diff --git a/player/loadfile.c b/player/loadfile.c
index 4890245..2b88ecf 100644
--- a/player/loadfile.c
+++ b/player/loadfile.c
@@ -452,7 +452,7 @@ void mp_switch_track_n(struct MPContext *mpctx, int order, enum stream_type type
         if (type == STREAM_VIDEO) {
             uninit_video_chain(mpctx);
             if (!track)
-                handle_force_window(mpctx, false);
+                handle_force_window(mpctx, true);
         } else if (type == STREAM_AUDIO) {
             clear_audio_output_buffers(mpctx);
             uninit_audio_chain(mpctx);
